_pytest/_pluggy.py

```diff
@@ -1,11 +1,6 @@
 """
-imports symbols from vendored "pluggy" if available, otherwise
-falls back to importing "pluggy" from the default namespace.
+Import symbols from ``pluggy``
```

Comment:
I think we can get rid of this module altogether and just import `pluggy` docs normally no?